multitenancy/tenant-manager-client/src/main/java/io/apicurio/multitenant/client/Auth.java

```diff
@@ -22,15 +22,15 @@
 import org.keycloak.common.VerificationException;
 import org.keycloak.common.util.Time;
 import org.keycloak.representations.AccessToken;
-import org.keycloak.representations.AccessTokenResponse;
 
 import java.util.HashMap;
 
 public class Auth {
 
     private final AuthzClient keycloak;
     private static final String BEARER = "Bearer ";
-    private AccessTokenResponse accessToken;
+    private AccessToken accessTokenParsed;
+    private String accessToken;
 
 
     public Auth(String serverUrl, String realm, String clientId, String clientSecret) {
```

Comment:
would it make sense to make the tenant manager client use the `Auth` interface from the `apicurio-registry- common` module ?  this way we have this logic in only one place

Reply:
I know that the logic is duplicated but I don't think it's a good idea to add the `apicurio-registry-common` module as a dependency to the `tenant-manager-client`. At some point, my idea is to extract all of this to a common, re-usable repo with utilities for rest-clients since right now we have 3 of them and growing and I don't want to replicate this over and over and over.

Reply:
Actually, a part of that work started with the latest refactor to the rest-client into a common + two separated modules.

Reply:
sounds good, thanks